lib/qiita/markdown/transformers/filter_iframe.rb

```diff
@@ -40,7 +41,10 @@ def node
         end
 
         def host_of(url)
-          Addressable::URI.parse(url).host if url
+          if url
+            port = URI.parse(url).port
+            Addressable::URI.parse(url).host if [443, 80].include? port
```

Comment:
📝 Exclude requests other than http and https

Reply:
I think it needs test.